hack/tf-fmt.sh

```diff
@@ -9,6 +9,6 @@ else
     --env IS_CONTAINER=TRUE \
```

Comment:
Above, in #L6, please update `/terraform` to `terraform`  i've pushed an 11.8 terraform image using the change you made to the build script-adding terraform binary to PATH, so now you  `terraform fmt...` no leading slash.  
You can verify image is available, `docker pull quay.io/coreos/terraform-alpine:v0.11.8`  or you can run `./hack/tf-fmt.sh` from installer dir to verify all is working well (I did, it is :+1: ).

Reply:
> Above, in #L6, please update `/terraform` to `terraform`...

Done and rebased onto master with 59af76b -> 0032348.

Reply:
ack yup i missed that